README.md

```diff
@@ -1,168 +1,95 @@
-
-![Creator logo](creatorlogo.png)
+[![Creator logo](creatorlogo.png)](https://community.imgtec.com/platforms/creator-ci40/)
```

Comment:
Use the new logo https://github.com/CreatorKit/creator-docs/blob/master/images/creatorlogo.png